libs/sr/solver/src/childWorker.ts

```diff
@@ -120,17 +124,23 @@ async function start() {
                 rope.stats,
               ])
 
-              builds.push({
-                value: results[0], // We only pass 1 target to calculate, so just grab the 1st result
-                ids: {
-                  head: head.id,
-                  hands: hands.id,
-                  feet: feet.id,
-                  body: body.id,
-                  sphere: sphere.id,
-                  rope: rope.id,
-                },
-              })
+              if (
```

Comment:
This messes up the calculated build count, since that is handled in sliceSortSendBuilds(), which occurs after this